lib/embulk/output/bigquery/bigquery_client.rb

```diff
@@ -331,9 +331,11 @@ def wait_load(kind, response)
           end
 
           # cf. http://www.rubydoc.info/github/google/google-api-ruby-client/Google/Apis/BigqueryV2/JobStatus#errors-instance_method
-          # `errors` returns Array<Google::Apis::BigqueryV2::ErrorProto> if any error exists.
+          # `error_result` returns Google::Apis::BigqueryV2::ErrorProto if job failed.
           # Otherwise, this returns nil.
-          if _errors = _response.status.errors
+          if _response.status.error_result
+            # `errors` returns Array<Google::Apis::BigqueryV2::ErrorProto> if any error exists.
+            _errors = _response.status.errors
```

Comment:
Don’t you want warning log to show errors if it exists when error_resulst is nil (success)?

Reply:
Thank you for your review! That's a good point. I'll change it.